### PR TITLE
Fix build after moving opener from AB CLI into the common lib

### PR DESCRIPTION
**Pull request by teobugslayer**

### CODE REVIEW

**Comment by rosen-vladimirov** [comment]
:+1:  after green build

File: lib/declarations.d.ts [hunk 1]
```diff
@@ -162,7 +162,7 @@ interface IProjectCapabilities {
 }
 
 interface IOpener {
-	open(filename: string): void;
+	open(filename: string, appname?: string): void;
```
Comment (Fatme) [inline lib/declarations.d.ts]: Shouldn't this interface be in common lib?

**Comment by Fatme** [comment]
You should also remove opener from bootstrap.

File: lib/declarations.d.ts (hunk 1 above)
Comment (teobugslayer) [inline lib/declarations.d.ts]: Done! Nice catch!

**Pull request merged**